docs/apache-airflow/executor/kubernetes.rst

```diff
@@ -46,26 +46,26 @@ This command generates the pods as they will be launched in Kubernetes and dumps
 pod_template_file
 #################
 
-As of Airflow 1.10.12, you can now use the ``pod_template_file`` option in the ``kubernetes`` section
+As of Airflow 1.10.12, you can use the ``pod_template_file`` option in the ``kubernetes`` section
```

Comment:
```suggestion
As of Airflow 2.0.0, you now use the ``pod_template_file`` option in the ``kubernetes`` section
```

Might as well rip off the bandaid - docs are versioned after all.

Reply:
I think the intention of this sentence is to say when the feature was added, so if we want to leave it structured like this I think it makes sense to leave 1.10.12.  However, at a certain point, it doesn't matter when the feature was added.  So we _could_ just remove that reference and get to the point --- "To customize the pod used for k8s executor worker processes, _use_ `pod_template_file`..." or similar.  WDYT?

Reply:
Yeah, lets do that. Might be able to get rid of the last sentence in this paragraph too then - it made more sense when both this and the airflow.cfg approach was supported.